crowbar_framework/app/views/nodes/_buttons_power.html.haml

```diff
@@ -1,10 +1,23 @@
-- if @node.admin?
+.btn-group{ role: "group" }
```

Comment:
Please remove this, it's already added here: https://github.com/jsuchome/barclamp-crowbar/blob/buttons/crowbar_framework/app/views/nodes/_show.html.haml#L23
I think after that the width of the dropdown is better.